src/js/controller/controller.js

```diff
@@ -289,7 +289,7 @@ define([
                 if (_model.get('playOnViewable')) {
                     if (viewable) {
                         _autoStart();
-                    } else if (utils.isMobile()) {
+                    } else if (utils.isMobile() && model.get('mute')) {
```

Comment:
I'm not sure we want to use the mute setting for this. I thought when we use 'playOnViewable' there is a second mute setting that indicates that autostart mute is active. 'mute' just indicates the users preference, not whether or not we're dealing with this state where we are muted because of 'playOnViewable'.